.github/workflows/lint-oas.yml

```diff
@@ -10,21 +10,5 @@ on:
   workflow_dispatch:
 
 jobs:
-  run:
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        component: ['autorisaties', 'besluiten', 'catalogi', 'documenten', 'zaken']
-
-    name: Run with component ${{ matrix.component }}
-
-    steps:
-      - uses: actions/checkout@v4
-      - name: Use Node.js
-        uses: actions/setup-node@v4
-        with:
-          node-version: '18'
-      - name: Install spectral
-        run: npm install -g @stoplight/spectral@5
-      - name: Run OAS linter
-        run: spectral lint ./src/openzaak/components/${{ matrix.component }}/openapi.yaml
```

Comment:
I wonder if you can define the matrix in this fine and keep the old structure in  open-api-workflows instead of using the bash script?